e2e/utils_test.go

```diff
@@ -0,0 +1,25 @@
+package e2e
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDownloadFile(t *testing.T) {
```

Comment:
@jsoriano I think the error on CI comes from an issue reading the downloaded file. I've added this test for it, but did not have time to run it on CI. Could you please check it out in your morning? We can talk about it later on tomorrow.

Thanks in advance!

Reply:
BTW, I think it happened on your local machine too, could you please run this test locally?